recipe/bld.bat

```diff
@@ -11,5 +11,5 @@ ECHO toolkits_tests = False >> setup.cfg
 ECHO [test] >> setup.cfg
 ECHO local_freetype = True >> setup.cfg
 
-%PYTHON% setup.py install --single-version-externally-managed --record record.txt
+%PYTHON% -m pip install -v 
```

Comment:
I believe that `python -m pip install --no-deps --ignore-installed .` is safer.

Reply:
Seems fair. I'm just trying to get the damned thing to work.